host-os-iso-creator.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) IBM Corp. 2016.
```

Comment:
PEP8 is against using dashes in file names because they cannot be imported with a standard import statement. I suggest create_host_os_iso.py
I think we should rename the host-os-build.py file too, but I don't know if we can do this now with the backwards compatibility issue.